static/js/admin/src/ExportMenu.js

```diff
@@ -0,0 +1,101 @@
+define('ExportMenu', [ 'jquery', 'jquery-ui', 'underscore', 'qtip', 'cycle'],
+    function($, ui, _) {
+
+        function ExportMenu(options) {
+            this.options = $.extend({}, options);
+            this.init();
+            this.refresh();
+        }
+
+        ExportMenu.prototype.getMenuItems = function() {
+            let items = {};
+            let targets = this.getTargets();
+
+            if (this.getTargetIds() !== '') {
+                items["exportMemberOrder"] = {name : "Member Order"};
+            }
+            if ($.inArray('editDescription', targets[0].metadata.permissions) !== -1) {
+                items["exportXML"] = {name : "Bulk Metadata"};
+            }
+
+            return items;
+        };
+
+        ExportMenu.prototype.getTargetIds = function() {
+            let ids = this.getTargets().filter((d) => {
+                return $.inArray("viewHidden", d.metadata.permissions) !== -1 && d.metadata.type === "Work"
+            }).map(d => d.metadata.id);
+            return ids.join(',');
+        }
+
+        ExportMenu.prototype.canEditDescription = function() {
+            return $.inArray('editDescription', this.getTargets()[0].metadata.permissions) !== -1;
+        }
+
+        ExportMenu.prototype.getTargets = function() {
+            return JSON.retrocycle(JSON.parse(sessionStorage.getItem('exportTargets')));
+        }
+
+        ExportMenu.prototype.refresh = function() {
+            let items = this.getMenuItems();
+            if ($.isEmptyObject(items)) {
+                $(this.options.selector).hide();
+            } else {
+                $(this.options.selector).show();
+            }
+        }
+
+        ExportMenu.prototype.init = function() {
+            let self = this;
+
+            this.menu = $.contextMenu({
+                selector: this.options.selector,
+                trigger: 'left',
+                build: function($triggerEvent, e) {
+                    let items = self.getMenuItems();
+
+                    return {
+                        className: 'add_to_container_menu',
+                        events : {
+                            show: function() {
+                                this.addClass("active");
+                            },
+                            hide: function() {
+                                this.removeClass("active");
+                                sessionStorage.clear();
```

Comment:
Will this clear everything in the sessionStorage for the dcr? Would it make sense to just clear the `exportTargets` field?